src/engine/world/world.cpp

```diff
@@ -1070,37 +1072,58 @@ void World::save(const std::filesystem::path& filename, bool isQuicksave)
   doc.save("meta", meta, meta);
   doc.save("data", *this, *this);
   doc.write();
+
+  if(metaPath.has_value())
+  {
+    serialization::YAMLDocument<false> metaCacheDoc{metaPath.value()};
+    metaCacheDoc.save("meta", meta, meta);
+    metaCacheDoc.write();
+  }
```

Comment:
The extension replacement should be extracted into a utility function like `makeMetaFilepath(saveFilepath)`, but this is the idea:
```suggestion
  auto metaPath = filename;
  metaPath.replace_extension(".meta.yaml");
  serialization::YAMLDocument<false> metaCacheDoc{metaPath};
  metaCacheDoc.save("meta", meta, meta);
  metaCacheDoc.write();
```

The utility function would also guarantee that the meta file isn't in a completely different path than the actual save file.